paraatm/tests/vcas_example.py

```diff
@@ -0,0 +1,55 @@
+#from vcasmodule import VCAS
+import os
+import numpy as np
+from paraatm.plotting import plot_trajectory
+from paraatm.simulation_method.vcas import VCAS
+import matplotlib.pyplot as plt
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+data_dir = os.path.join(cur_dir, '..', 'sample_data/')
+# set environmental variable NATS_HOME
+os.environ["NATS_HOME"] = "/home/ywang542/NATS/NATS_1.7"
```

Comment:
Please remove this reference to your local NATS installation.  In general, please do not set `NATS_HOME` as part of the test cases.  Every user will have a different `NATS_HOME`, so they will need to set this themselves, outside of para-atm.

Reply:
I found that the environmental variable "NATS_HOME" should be assigned for my code to run. Do you have a solution to this?

Reply:
Set it outside of para-atm on the command line:
```shell
export NATS_HOME=/my/path
```
If you want this setting to be persistent, add it to your `~/.bashrc` file.